pset-functions.js

```diff
@@ -136,6 +183,53 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const pointChart = [
+  { letters: 'aeioulnrst', points: 1 },
+  { letters: 'dg', points: 2 },
+  { letters: 'bcmp', points: 3 },
+  { letters: 'fhvwy', points: 4 },
+  { letters: 'k', points: 5 },
+  { letters: 'jx', points: 8 },
+  { letters: 'qz', points: 10 }
+]
+
+function scrabbleScore (word) {
+  const lettersInWord = word.split('')
+  let wordScore = 0
+
+  //  iterate over the letters in the submitted word
+  lettersInWord.forEach(function (letter) {
+    // interate over the points array and find which letter
+    // is currently selected
+    pointChart.forEach((value) => {
+      // debug
+      // console.log(value['letters'])
+
+      //  create an array of letters that align the current
+      //  amount of points to be given. these letters will be compared
+      //  against the current letter in the word that was submitted
+      const comparisonLetters = value.letters.split('')
+
+      //  debug
+      //  console.log(comparisonLetters)
+
+      //  find if the current letter from the submitted word exists
+      //  within the set of letters for this point value.
+      //  if it does, add points. if not, the set the loop proceeds
+      //  to the next set of letters associated with value['points']
+      if (comparisonLetters.find(element => element === letter)) {
+        wordScore += (value.points)
+
+        // debug
+        // console.log(`new wordScore is ${wordScore}`)
+      }
+    })
+  })
+
+  return (`This word is worth ${wordScore} points`)
+}
+
+console.log(scrabbleScore('function'))
```

Comment:
@sad3xoxo Good Job here with this solution. I see you grouped the letters and values in an array of objects. This is a good way to structure your data for an example like this.
Take a look at this approach [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174)